src/Core/Migrate/Migrate.php

```diff
@@ -55,7 +57,7 @@ public static function migrateSettingsFile(Event $event = null)
                     'mailer_host'       => '127.0.0.1',
                     'mailer_user'       => '~',
                     'mailer_password'   => '~',
-                )
+                ) + $default_parameters['parameters']
```

Comment:
Shouldn't be the other way around ? Default being overwritten by legacy values ?